services/data-collection/user-event.mdx

```diff
@@ -4,13 +4,36 @@ description: Detailed `user` event guide
 icon: dash
 ---
 
-The `user` event is a fundamental aspect of user analytics, enabling you to connect users to their actions and 
-capture essential properties about them. This documentation provides detailed guidance on utilizing the `user` event to enrich 
+The `user` event is a fundamental aspect of user analytics, enabling consistent tracking and reference across their lifetime. 
+Whether you’re dealing with registered users or anonymous visitors, the `user` event helps you associate meaningful data with each individual.  
+This documentation provides detailed guidance on utilizing the `user` event to enrich 
 your user data for more personalized and insightful analytics.
 
 First of all, you don't need to fire an `user` event for every user action. Instead, you should fire it when 
 you first learn about a user, and then update the user's information as it changes (for example, when they update their profile, or when they first register).
 
+## How It Works  
+
+When you fire a `user` event using Edgee’s JavaScript SDK or Data Layer, Edgee securely processes and stores the user data at the edge. 
```

Comment:
Do we actually store the user data at the edge? 🤔 

That sounds bad. Or do we just store some of it in a cookie so we can use it for all subsequent events?

If that's the case, I'd like to find a better way to say this, like:
> When you fire a `user` event using the Edgee SDK or Data Layer, we securely process the user data at the edge and make sure all subsequent events are bound to that user.